index.html

```diff
@@ -0,0 +1,273 @@
+                        industry's standard.</p>
+                    <ul class="grid__item-stack">
+                        <li class="grid__bootstrap">Bootstrap</li>
+                        <li class="grid__html">HTML</li>
+                        <li class="grid__ruby">Ruby</li>
+                    </ul>
+                    <button class="grid__item-button">See project</button>
+                </div>
+                <div class="grid__item">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul class="grid__item-stack">
+                        <li class="grid__bootstrap">Bootstrap</li>
+                        <li class="grid__html">HTML</li>
+                        <li class="grid__ruby">Ruby</li>
+                    </ul>
+                    <button class="grid__item-button">See project</button>
+                </div>
+                <div class="grid__item">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul class="grid__item-stack">
+                        <li class="grid__bootstrap">Bootstrap</li>
+                        <li class="grid__html">HTML</li>
+                        <li class="grid__ruby">Ruby</li>
+                    </ul>
+                    <button class="grid__item-button">See project</button>
+                </div>
+                <div class="grid__item">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul class="grid__item-stack">
+                        <li class="grid__bootstrap">Bootstrap</li>
+                        <li class="grid__html">HTML</li>
+                        <li class="grid__ruby">Ruby</li>
+                    </ul>
+                    <button class="grid__item-button">See project</button>
+                </div>
+                <div class="grid__item">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul class="grid__item-stack">
+                        <li class="grid__bootstrap">Bootstrap</li>
+                        <li class="grid__html">HTML</li>
+                        <li class="grid__ruby">Ruby</li>
+                    </ul>
+                    <button class="grid__item-button">See project</button>
+                </div>
+                <div class="grid__item">
```

Comment:
- [ ] A very big thumbs up to you 👍🏽 , your designs looks really good. One thing left is for you to implement the hover state of each project card. 

Kindly note that the default styling of all the cards is this 👇🏽 
![image](https://github.com/sinruidotec/Desktop-Version-Portfolio/assets/63473584/c1aabf74-45f7-4ebc-ac2a-8f59435ccdcd)

Now when you hover over a project card that's when it will show the See Project button and the image of that particular project like this 👇🏽 
![image](https://github.com/sinruidotec/Desktop-Version-Portfolio/assets/63473584/d67ba1b7-58cb-45f9-a55e-76f1cf279970)

see further explanation below 👇🏽 
[See explanation here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4?node-id=34:1278&mode=dev#407198889)

Reply:
I wanted to thank you for the suggestions, it's gratifying to learn from code reviews, this was a big challenge for me as I had some trouble getting the desired result.

I also learned a new way to use Figma thanks to the link that took me directly to the design comments.